### ISSUE TITLE
Core - ExecuteScriptAsync(script) incorrectly appends braces and semicolon to script.

### ISSUE THREAD

**Issue by amaitland** [cefsharp/CefSharp#3944]
**What version of the product are you using?**
96.0.180

**What architecture x86 or x64?**
Any

**What version of .Net?**
Any

**On what operating system?**
Win10

**Are you using WinForms, WPF or OffScreen?**
Any

**What steps will reproduce the problem?**

```c#
chromiumWebBrowser.ExecuteScriptAsync("document.getElementsByTagName('input')[2].click();")
```

The executed script is actually the following which causes an error
```js
document.getElementsByTagName('input')[2].click();();
```

**Comment by amaitland** [comment]
Additional details at https://github.com/cefsharp/CefSharp/discussions/3943#discussioncomment-1910574

**Comment by amaitland** [comment]
Using the following should work as expected and is functionality the same (JavaScript is executed on the main frame by default).

```c#
chromiumWebBrowser.GetMainFrame().ExecuteJavaScriptAsync(script); 
```

**Comment by Drmsingh** [comment]
> Using the following should work as expected and is functionality the same (JavaScript is executed on the main frame by default).
> 
> ```cs
> chromiumWebBrowser.GetMainFrame().ExecuteJavaScriptAsync(script); 
> ```

Ok, I have tried as provided workaround, it works and going to implement this in our enterprises app. So you will apply the same fix in upcoming cefsharp 97 or required any more changes with this workaround.
After applied, will any impact in the application to need take care other behavior?

**Comment by amaitland** [comment]
The workaround will continue to work going forward. No further changes will be nessicary.

**Comment by amaitland** [comment]
Fix is available in [97.1.10-pre](https://github.com/cefsharp/CefSharp/issues/3956#issuecomment-1009571154) which will be available on `Nuget.org` shortly.

**Comment by davidei1955** [comment]
I had an apparently related problem after I upgraded my WinForms app from CefSharp v86.0.241.0 to v96.0.180.0.
I was seeing:  _"Uncaught SyntaxError: Unexpected token ')'", source: about:blank (1)_ in the javascript console after loading web pages with:    `CefBrowser.LoadHtml(html-page-as-string, dummyUrl);`
Upgrading again from CefSharp v96.0.180.0 to v97.1.10-pre has resolved the problem. Thanks!
